src/renderer/components/editor.vue

```diff
@@ -2,7 +2,8 @@
   <div
     class="editor-wrapper"
     :class="[{ 'typewriter': typewriter, 'focus': focus, 'source': sourceCode }, theme]"
-    :style="{ 'color': theme === 'dark' ? darkColor : lightColor, 'lineHeight': lineHeight, 'fontSize': fontSize, 'font-family': editorFontFamily }"
+    :style="{ 'color': theme === 'dark' ? darkColor : lightColor, 'lineHeight': lineHeight, 'fontSize': fontSize, 
+    'font-family': editorFontFamily ? `${editorFontFamily}, sans-serif`:'sans-serif'}"
```

Comment:
I think you need to judge weather `editorFontFamily ` is support be Chrome in Electron? not just judge `editorFontFamily ?`

Reply:
Yeap! I know that. But after refer other project like VSCode and google searching, I still can't find a good way to do that.
Something confuse me like where can I get the supported and unsupported list in electron(base on Chromium).
Is all what I need to do is check if user input's font-family in [Google font directory](https://fonts.google.com/)?
Or I have to write some tricks code to check is it work on chrome? Like use [CSS.supports()](https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports) or something else?
Here you can check how VSCode(which also develop base electron)do about this: 
[Related test code](https://github.com/Microsoft/vscode/blob/165686e333508bc838d580b3738292d7fa9b82d0/src/vs/workbench/parts/terminal/test/electron-browser/terminalConfigHelper.test.ts#L36)
[Related code]
(https://github.com/Microsoft/vscode/blob/17ed2280597e68fefa00b43c8d28d0442891ea22/src/vs/workbench/parts/terminal/electron-browser/terminalConfigHelper.ts#L91)
I still can't figure out how they do something like misspells check or font-family support check in electron(base on Chromium) , and what is the better way to do that.
Please give some help! Thanks!

Reply:
Do you think that `fallback` font family is enough?

Reply:
Is there any best best practices to implement that ?
I am also take it importance but have no idea about how to implement it in proper way.